src/Client/SidebarComponents/Navbar.fs

```diff
@@ -67,11 +51,10 @@ let createMetaDataTypeButtons excelMetadataType setExcelMetadataType (dispatch:
                     let investigation = ArcInvestigation.init("New Investigation")
                     setExcelMetadataType {
                         excelMetadataType with
-                            Identifier = Some WorkSheetNames.investigation
-                            MetadataType = Some ArcFilesDiscriminate.Investigation
-                            Investigation = Some investigation
+                            WorkSheetName = Some ArcInvestigation.metadataSheetName
+                            TopLevelMetadata = Some (ArcFiles.Investigation investigation)
                     }
-                    OfficeInterop.CreateTopLevelMetadata(WorkSheetNames.investigation)
+                    OfficeInterop.CreateTopLevelMetadata(ArcInvestigation.metadataSheetName)
```

Comment:
this is still not backwards compatible. You must use the isMetadataSheet function

Reply:
fixed